microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -175,42 +201,41 @@ To pause on the first line of the top function in your call stack, while paused
 The following code is an example for you to step through:
 
 ```javascript
-function factorial(n) {
-    var product = 0; // B
-    for (var i = 1; i <= n; i++) {
-      product += i;
-    }
-    return product; // A
+function updateLabel() {
+  var addend1 = getNumber1(); // B
+  var addend2 = getNumber2();
+  var sum = addend1 + addend2;
+  label.textContent = addend1 + " + " + addend2 + " = " + sum; // A
 }
 ```
 
-You are paused on `A`.  After selecting **Restart frame**, you should be paused on `B`, without ever setting a breakpoint or choosing **Resume script execution**.
+You are paused on `A`.  After selecting **Restart frame**, you're be paused on `B`, without having to set a breakpoint on that line and choosing **Resume script execution**.
 
-![Selecting Restart frame](./reference-images/javascript-source-page-debugger-restart-frame.png)
+![Selecting Restart frame](./reference-images/restart-frame.png)
 
 
 <!-- ------------------------------ -->
 #### Resume script runtime
```

Comment:
this heading is meaningless to me.  nonstandard wording?  can we express in relatable terms; what would classroom instructor say to students here?

Reply:
fixed
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/reference?branch=pr-en-us-2894#resume-script-execution